sdk/servicebus/azure_messaging_servicebus/CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+# Running Azure Service Bus Live Tests
+
+This guide explains how to set up and run live integration tests for the Azure Service Bus SDK for Rust using the Azure SDK test infrastructure.
+
+## Prerequisites
+
+1. **Azure Subscription**: Access to the targeted subscription
+2. **Azure CLI**: Install and configure Azure CLI (`az login`)
+3. **Azure PowerShell**: Install PowerShell modules (`Install-Module Az -Force`)
+4. **Bicep**: For infrastructure deployment (`az bicep install`)
+5. **Permissions**: Contributor access to create resources in your subscription
+
+## Automated Test Infrastructure Setup
+
+The Azure SDK provides a standardized test infrastructure using the `New-TestResources.ps1` script with Bicep templates to automatically create and configure all required Azure Service Bus resources.
+
+### Deploy Test Resources
+
+From the repository root, run:
+
+```powershell
+# Set subscription
+az account set --subscription "<Your Subscription Name>"
+
+# Deploy test infrastructure
+./eng/common/TestResources/New-TestResources.ps1 servicebus/azure_messaging_servicebus -SubscriptionId <subscription ID>
+```
+
+This will automatically create:
+
+- Azure Service Bus namespace with Standard tier
+- Test queue named `testqueue`
+- Test topic named `testtopic` with subscription `testsubscription`
+- RBAC permissions for TokenCredential testing
+- All required environment variables
+
+The script will output PowerShell commands to set environment variables.
+
+### Environment Variables
+
+After deployment completes successfully, the script will output commands like:
+
+```powershell
+$env:SERVICEBUS_NAMESPACE = "sb-your-deployment-name.servicebus.windows.net"
+$env:SERVICEBUS_QUEUE_NAME = "testqueue"
+$env:SERVICEBUS_TOPIC_NAME = "testtopic"
+$env:SERVICEBUS_SUBSCRIPTION_NAME = "testsubscription"
+```
```

Comment:
Not true. We write .env files. I'd just get rid of this file. Not even EH - which is unique - has its own contrib file. These are just duplication - and incorrect - of what we already have in the root that applies to all crates.